src/main/java/org/snomed/snowstorm/core/data/services/ConceptService.java

```diff
@@ -56,6 +56,14 @@
 
 @Service
 public class ConceptService extends ComponentService {
+	private static final Map<ComponentType, Class<? extends DomainEntity<?>>> DOCUMENTS = new HashMap<>();
```

Comment:
The logic for querying Concepts, Descriptions, Relationships and Axioms are pretty much the identical - using a collection allowed me to iterate and stop repeating code.

Reply:
Could you give this a more specific name like `COMPONENT_DOCUMENT_TYPES`?

Reply:
Good suggestion, I have done this.